src/overlays/actors/ovl_En_Rz/z_en_rz.c

```diff
@@ -15,21 +16,39 @@ void EnRz_Destroy(Actor* thisx, PlayState* play);
 void EnRz_Update(Actor* thisx, PlayState* play);
 void EnRz_Draw(Actor* thisx, PlayState* play);
 
-void func_80BFC058(EnRz* this, PlayState* play);
+void EnRz_ActorShadowFunc(Actor* thisx, Lights* mapper, PlayState* play);
+void EnRz_ChangeAnim(PlayState*, EnRz*, s16 animIndex, u8 animMode, f32 transitionRate);
+s32 EnRz_SetupPath(EnRz* this, PlayState* play);
+EnRz* EnRz_FindSister(EnRz* this, PlayState*);
+void EnRz_SetupUpdateSkelAnime(EnRz* this, PlayState* play);
 void func_80BFC078(EnRz* this, PlayState* play);
-void func_80BFC19C(EnRz* this, PlayState* play);
-void func_80BFC214(EnRz* this, PlayState* play);
-void func_80BFC270(EnRz* this, PlayState* play);
-void func_80BFC2F4(EnRz* this, PlayState* play);
-void func_80BFC36C(EnRz* this, PlayState* play);
 void func_80BFC3F8(EnRz* this, PlayState* play);
-void func_80BFC608(EnRz* this, PlayState* play);
 void func_80BFC674(EnRz* this, PlayState* play);
-void func_80BFC728(EnRz* this, PlayState* play);
 void func_80BFC7E0(EnRz* this, PlayState* play);
-void func_80BFC8F8(EnRz* this, PlayState* play);
+void EnRz_Walk(EnRz* this, PlayState* play);
+
+#define EN_RZ_STATE_1 (1 << 0)
+#define EN_RZ_STATE_2 (1 << 1)
+
+typedef enum {
+    /* 0 */ EN_RZ_ANIM_THINKING,
+    /* 1 */ EN_RZ_ANIM_STANDING,
+    /* 2 */ EN_RZ_ANIM_WALKING,
+    /* 3 */ EN_RZ_ANIM_SITTING,
+    /* 4 */ EN_RZ_ANIM_APPLAUDING,
+    /* 5 */ EN_RZ_ANIM_ON_KNEES,
+    /* 6 */ EN_RZ_ANIM_DANCE,
+    /* 7 */ EN_RZ_ANIM_LINK_NORMAL_WAIT_FREE,
+    /* 8 */ EN_RZ_ANIM_LINK_DANCE,
+    /* 9 */ EN_RZ_ANIM_MAX
+} EnRzAnimations;
```

Comment:
```suggestion
} RosaSistersAnimation;
```
Would prefer sticking with 1 prefix for all the enums, either `EnRz` or `RosaSisters`. If you do change it, don't forget to actually update the prefixes of the enum values. Also enum names should be singular.